portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
```

Comment:
nit: When building a REST API, URIs should generally denote the collection of resources you're operating on: https://restfulapi.net/resource-naming/. In this case, something like "inquiries" or "posts" might be appropriate, depending on how you're thinking about what the user is creating by submitting a filled-out form.

Reply:
Thanks for sharing the link, I did not know about the distinctions made in the reference. I think I was looking at it as "the URI points to the component".

I will change the URI to "posts".